examples/advanced/main.tf

```diff
@@ -142,9 +109,9 @@ module "cos_bucket1" {
   management_endpoint_type_for_bucket = var.management_endpoint_type_for_bucket
   existing_kms_instance_guid          = module.key_protect_all_inclusive.kms_guid
   kms_key_crn                         = module.key_protect_all_inclusive.keys["${local.key_ring_name}.${local.key_name}"].crn
-  monitoring_crn                      = module.observability_instances.cloud_monitoring_crn
   retention_enabled                   = false # disable retention for test environments - enable for stage/prod
-  activity_tracker_crn                = local.at_crn
+  activity_tracker_read_data_events   = false # disable activity_tracker
+  activity_tracker_write_data_events  = false # disable activity_tracker
```

Comment:
There is no need to set these to false - just remove them and allow the default to be used